src/counter.rs

```diff
@@ -326,7 +326,10 @@ mod tests {
     use std::f64;
 
     use super::*;
-    use crate::metrics::{Collector, Opts};
+    use crate::{
+        metrics::{Collector, Opts},
+        proto_ext::MessageFieldExt,
```

Comment:
This one seems to be missing a `#[cfg(feature = "protobuf")]`, I guess.

I think other test modules may also be broken in the same way under `--no-default-features`.

Reply:
thanks, fixed now with bf6c3c1927c830640084ce7445ead32320240923